library/std/src/thread/mod.rs

```diff
@@ -337,6 +350,61 @@ impl Builder {
         self
     }
 
+    /// **armv6k-nintendo-3ds specific!**
+    ///
+    /// Sets the priority level for the new thread.
+    ///
+    /// Low values gives the thread higher priority. For userland apps, this has
+    /// to be within the range of 0x18 to 0x3F inclusive. The main thread usually
+    /// has a priority of 0x30, but not always.
+    ///
+    /// # Examples
+    /// ```
+    /// use std::thread;
+    ///
+    /// let builder = thread::Builder::new().priority(0x30);
+    /// ```
+    #[cfg(target_os = "horizon")]
+    #[stable(feature = "horizon_thread_ext", since = "1.60.0")]
+    pub fn priority(mut self, priority: i32) -> Builder {
+        self.priority = Some(priority);
+        self
+    }
+
+    /// **armv6k-nintendo-3ds specific!**
+    ///
+    /// Sets the ID of the processor the thread should be run on.
+    ///
+    /// Processor IDs are labeled starting from 0. On Old3DS it must be <2, and
+    /// on New3DS it must be <4. Pass -1 to execute the thread on all CPUs and
+    /// -2 to execute the thread on the default CPU (set in the application's Exheader).
+    ///
+    /// *Processor #0 is the application core. It is always possible to create a thread on this
+    /// core.
+    /// *Processor #1 is the system core. If APT_SetAppCpuTimeLimit is used, it is possible
+    /// to create a single thread on this core.
+    /// *Processor #2 is New3DS exclusive. Normal applications can create threads on
+    /// this core if the exheader kernel flags bitmask has 0x2000 set.
+    /// *Processor #3 is New3DS exclusive. Normal applications cannot create threads
+    /// on this core.
+    ///
+    /// Processes in the BASE memory region can always create threads on
+    /// processors #2 and #3.
```

Comment:
This and the documentation about the core#3 aren’t necessary. This explanation is outside Rust’s std scope and only really matter when programming the CFW. Just for the sake of simplicity.

Reply:
Sure, I just copied these docs from the ctru-rs module, but sounds like they need some changes.